shared/tasks/components.mjs

```diff
@@ -44,7 +43,7 @@ export async function generateFixtures (pattern, { srcPath, destPath }) {
  * Generate macro-options.json from component data
  *
  * @param {AssetEntry[0]} pattern - Path to ${componentName}.yaml
- * @param {AssetEntry[1]} options - Asset options
+ * @param {Pick<AssetEntry[1], "srcPath" | "destPath">} options - Asset options
```

Comment:
This task already provides its own `filePath` and `fileContents`

Here we're narrowing down the types to _warn_ when passing in options that aren't picked